### PR TITLE
feat: add url feature

### PR DESCRIPTION
**Pull request by foolishell**
for issue: https://github.com/cksac/fake-rs/issues/189

I think it's best to use https://example.com/ as a fixed URL.

### CODE REVIEW

File: fake/src/impls/url/mod.rs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+use crate::{Dummy, Fake, Faker};
+use rand::Rng;
+use url::Url;
+
+const FQDN: &str = "https://example.com";
+
+impl Dummy<Faker> for Url {
+    fn dummy_with_rng<R: Rng + ?Sized>(_: &Faker, rng: &mut R) -> Self {
+        let path: String = Faker.fake_with_rng(rng);
```
Comment (cksac) [inline fake/src/impls/url/mod.rs:9]: are arbitrary string can be a valid path?

File: fake/src/impls/url/mod.rs [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+        Url::parse(&format!("{FQDN}/{path}")).unwrap()
```
Comment (cksac) [inline fake/src/impls/url/mod.rs:10]: the unwrap may panic due to path is not valid, I think we should make it alway valid path?

File: fake/src/impls/url/mod.rs (hunk 1 above)
Comment (foolishell) [reply fake/src/impls/url/mod.rs]: I think it is. If path is `[a-zA-Z]`, it must be valid.

However, I think it's better to define paths as const.
FIXED: [c736536](https://github.com/cksac/fake-rs/pull/190/commits/c736536a9b80d557bd52662aab13e1cea2086ea3)

**Comment by foolishell** [comment]
@cksac 
Hi, could you review again?
And I have no idea why Clippy is failed. Could you help it?

**Comment by cksac** [comment]
LGTM. will fix the Clippy warning later

**Comment by foolishell** [comment]
Thank you @cksac !
Please release the new version at your earliest convenience.

**Pull request merged**